tests/Integration/Http/ResourceTest.php

```diff
@@ -553,7 +553,7 @@ public function testResourcesMayCustomizeJsonOptionsOnPaginatedResponse()
         );
 
         $this->assertEquals(
-            '{"data":[{"id":5,"title":"Test Title","reading_time":3.0}],"links":{"first":"\/?page=1","last":"\/?page=1","prev":null,"next":null},"meta":{"current_page":1,"from":1,"last_page":1,"links":[{"url":null,"label":"&laquo; Previous","active":false},{"url":"\/?page=1","label":"1","active":true},{"url":null,"label":"Next &raquo;","active":false}],"path":"\/","per_page":15,"to":1,"total":10}}',
+            '{"data":[{"id":5,"title":"Test Title","reading_time":3.0}],"links":{"first":"\/?page=1","last":"\/?page=1","prev":null,"next":null},"meta":{"current_page":1,"from":1,"path":"\/","per_page":15,"to":1,"last_page":1,"links":[{"url":null,"label":"&laquo; Previous","active":false},{"url":"\/?page=1","label":"1","active":true},{"url":null,"label":"Next &raquo;","active":false}],"total":10}}',
```

Comment:
Because we are using array merge in the paginators to add custom data:

```php
public function toArray()
{
    return array_merge(parent::toArray(), [
        'next_page_url' => $this->nextPageUrl(),
     ]);
}
```

The keys can have different orders. So when encoding to JSON and the comparing it with a string, they won't match because some values can change place. This is what happened here.